### DISCUSSION TITLE
Multiple plurals in translation

### DISCUSSION THREAD

**Discussion by Kuldas** [statamic/cms#5587] [Q&A]
Hello, community, 

I'm creating a Czech translation (`cs`) and I've encountered a limitation that I don't know how to solve. I'll show it with an example for better understanding.

In `resource/lang/cs.json` on line `2` there is a translation for the number of available updates. There is an option to translate singular and plural using `|.` I totally understand this, but the problem is in the Czech.

In Czech, it should correctly be like this

* **Singular**
**en** - 1 update available
**cs** - 1 dostupná aktualizace

* **Plural**
**en** - :count updates available
**cs** - :count dostup<ins>**né**</ins> aktualiza<ins>**ce**</ins> (**if count 2 to 4)**
**cs** - :count dostup<ins>**ných**</ins> aktualiza<ins>**cí**</ins> (**if count 5 to unlimited**)

Perhaps the example makes it more understandable.

Is there any way to ensure that the correct translation is displayed based on the quantity in the plural?
 
**For example:**
```
{
    "1 update available|:count updates available": "1 dostupná aktualizace|:count(2-4) dostupné aktualizace|:count(5-∞) dostupných aktualizací",
}
```
Thank you for your reply. And if anything is not clear to you, I will try to explain it better.

**Comment by jackmcdade** [comment]
I don’t believe there’s any way to do that, there’s simply the “singular” and “plural” options. Is there a more verbose way to say “one or more than one” in Czech? It doesn’t need to be a word for word translation as long as the meaning is clear.

**Reply by Kuldas** [reply]
There would be quite a lot of alternatives in Czech. An alternative could be for example:

* **Singular**
**en** - 1 update available
**cs** - 1 dostupná aktualizace

* **Plural**
**en** - :count updates available
**cs** - dostupné aktualizace - :count

So I would only show that updates are available and what number of updates it. I'll try @JohnathonKoster option first - https://github.com/statamic/cms/discussions/5587#discussioncomment-2408661.

**Comment by JohnathonKoster** [comment]
You're syntax was close! Something like this should work:

```
"1 update available|:count updates available": "1 dostupná aktualizace|[2,4] dostupné aktualizace|[5,*] dostupných aktualizací",
```

The docs for Laravel's range syntax can be found here: https://laravel.com/docs/9.x/localization#pluralization

**Reply by jackmcdade** [reply]
Oh snap look at that. 👏

**Reply by Kuldas** [reply]
Thank you so much for this tip! I was really close 😄 

I'll try it and let you know.
